build-system/tasks/extension-helpers.js

```diff
@@ -163,6 +163,13 @@ function getExtensionsToBuild(preBuild = false) {
     return extensionsToBuild;
   }
   extensionsToBuild = DEFAULT_EXTENSION_SET;
+
+  /**
+   * `gulp performance-test` sets the `--extensions_from` flag dynamically
+   * for the document being tested, reload argv here to pick up changes.
```

Comment:
I'm not convinced this is a good idea. I'll post a comment where you're modifying this and suggest an alternative.